nengo_loihi/simulator.py

```diff
@@ -141,16 +141,173 @@ class Simulator(object):
+        ('test_probe.py:test_input_probe', "shape mismatch"),
+        ('test_probe.py:test_slice', "ObjView not handled properly"),
+        ('test_probe.py:test_update_timing', "probe bug?"),
+        ('test_solvers.py:test_nosolver*', "NoSolver bug"),
+
+        # reset bugs
+        ('test_neurons.py:test_reset*', "sim.reset not working correctly"),
+
+        # non-PES learning rules
+        ('test_learning_rules.py:test_unsupervised*',
+         "non-PES learning rules not implemented"),
+        ('test_learning_rules.py:test_dt_dependence*',
+         "non-PES learning rules not implemented"),
+        ('*voja*', "voja not implemented"),
+        ('test_learning_rules.py:test_custom_type',
+         "non-PES learning rules not implemented"),
+
+        # Nengo bug
+        ('test_simulator.py:test_entry_point',
+         "logic should be more flexible"),
+
+        # ensemble noise
+        ('test_ensemble.py:test_noise*', "ensemble.noise not implemented"),
+
+        # probe types
+        ('test_connection.py:test_dist_transform',
+         "probe type not implemented"),
+        ('test_connection.py:test_decoder_probe',
+         "probe type not implemented"),
+        ('test_probe.py:test_defaults', "probe type not implemented"),
+        ('test_probe.py:test_ensemble_encoders', "probe type not implemented"),
+
+        # probe.sample_every
+        ('test_integrator.py:test_integrator',
+         "probe.sample_every not implemented"),
+        ('test_oscillator.py:test_oscillator',
+         "probe.sample_every not implemented"),
+        ('test_ensemble.py:test_product*',
+         "probe.sample_every not implemented"),
+        ('test_neurons.py:test_dt_dependence*',
+         "probe.sample_every not implemented"),
+        ('test_probe.py:test_multiple_probes',
+         "probe.sample_every not implemented"),
+
+        # needs better place and route
+        ('test_ensemble.py:test_eval_points_heuristic*',
+         "max number of compartments exceeded"),
+        ('test_neurons.py:test_lif*', "idxBits out of range"),
+
+        # serialization / deserialization
+        ('test_cache.py:*', "model pickling not implemented"),
+        ('test_copy.py:test_pickle_model', "model pickling not implemented"),
+        ('test_simulator.py:test_signal_init_values',
+         "nengo.builder.Model instances not handled"),
+
+        # progress bars
+        ('test_simulator.py:test_simulator_progress_bars',
+         "progress bars not implemented"),
+
+        # utils.connection.target_function (deprecated)
+        ('utils/tests/test_connection.py*',
+         "target_function (deprecated) not working"),
+    ]
 
     def __init__(self, network, dt=0.001, seed=None, model=None,  # noqa: C901
-                 precompute=False, target=None):
+                 precompute=False, target=None, progress_bar=None):
         self.closed = True  # Start closed in case constructor raises exception
+        self._time = 0
```

Comment:
Where is this used?

Reply:
In the `time` property. If you create a `Simulator` instance but don't call `run`, doing `sim.time` gives an `AttributeError`.

Reply:
Ahh, I see, we had this attribute before, it just was not initialized. I think it should be initialized in `reset` where `_n_steps` is set to zero. Does that make sense?

Reply:
Added a fixup for this.